bottlecap

```diff
@@ -0,0 +1,103 @@
+#!/bin/sh
+
+# bottlecap, 'cause it's kinda like cork... get it?
+
+usage() {
+	echo "usage:"
+	echo "bottlecap [--dev] [--dry-run] [--runtime $runtime] [--build-dir $build_dir]  [--container $container] assemblerargs..."
+	echo "  --dev: rebuild and reinstall mantle, the build scripts, and the ostree-releng-scipts before running coreos-assembler"
+	echo "  --dry-run: do not run CoreOS Assembler; print what commands would be executed instead"
+	echo "  --runtime: (auto) specify whether to use podman or docker. Defaults to podman if it is in \$PATH"
+	echo "  --build-dir: Where to put build artifacts, defaults to the current directory"
+	echo "  --container: which coreos-assembler container to use"
+	echo "  --help: print this help message"
+}
+
+dev=0
+dryrun=0
+help=0
+runtime="podman"
+command -v podman > /dev/null || runtime="docker"
```

Comment:
what does the `command` command do? Did we switch from `which` for a reason?

Reply:
i have never seen that before.. nice. :+1:

Reply:
shellcheck complained about it

Reply:
interesting.. `command` is a builtin vs an external command so maybe that's why it's better